### ISSUE TITLE
gmt pscoast -= return -R0/0/0/0

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#2142]
By a change I typed `gmt pscoast -=` in the terminal, and get `-R0/0/0/0` returned.
```
$ gmt pscoast -=   
-R0/0/0/0
```

What's this? Is it some hidden feature or a bug?

**Comment by PaulWessel** [comment]
Looks like a bug but it obviously passes the parser...

**Comment by PaulWessel** [comment]
Looks like a bit of both, and maybe @joa-quim remembers some of this: In the mgd77 supplement we often were dealing with many thousands of files, to the point where wildcards like *.mgd77 would exceed what was possible (get too long list, of something).  So I implemented a "list of files" mechanism where you created a list of filenames, say, files.lis, and then gave =files.lis to the module.  This would mean "open files.lis and process all the files in the list".  Being a generally useful feature, I ported that to the GMT core; see lines 543+ in GMT_Create_Options in gmt_parse.c.  However, it looks like this was never documented yet a core GMT feature for a long time.

Your particular luck was to give -= which is not the same at all.  It is an invalid option, so each module's parse function sends it to 

`n_errors += gmt_default_error (GMT, opt->option);
`

which has these checks:

```
case '-': break;	/* Skip indiscriminently */
case '=': break;	/* Skip indiscriminently */
case '>': break;	/* Skip indiscriminently since dealt with internally */
case 'B': error += (GMT->common.B.active[GMT_PRIMARY] == false && GMT->common.B.active[GMT_SECONDARY] == false); break;
...
```

Turns out **-=**_list_ was an alternative option for =list, just like **->**_file_ is a valid mechanism for specifying an output file and **-<**_file_ for input.  This is mostly used via the API.  The problem with your -= is that it is not caught as a file list option with a missing file (error) but passes that check and then the gmt _default_error above is only given the option so does not know there are no args.  I will add that case to GMT_Parse_Common and see if that helps.
